recipe/meta.yaml

```diff
@@ -10,28 +10,23 @@ source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-  patches:
-    # Adds a patch to get the result of running the test suite.
-    # Please see PR submitted upstream for more details.
-    #
-    # https://github.com/luispedro/mahotas/pull/81
-    #
-    - PR_81.diff
 
 build:
-  number: 1
+  number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - mahotas-features = mahotas.features_cli:main
 
 requirements:
   build:
     - toolchain
     - python
     - setuptools
-    - numpy x.x
+    - numpy 1.11.*
```

Comment:
This change also needs to be followed by a re-rendering to update the CIs. PR ( https://github.com/conda-forge/mahotas-feedstock/pull/19 ) should do exactly that.